pkgs/tools/misc/aptly/default.nix

```diff
@@ -33,8 +33,8 @@ buildGoPackage {
 
   postInstall = ''
     rm $bin/bin/man
-    mkdir -p $out/etc/bash_completion.d
-    ln -s ${aptlyCompletionSrc}/aptly $out/etc/bash_completion.d
+    mkdir -p $out/share/bash-completion/completions
+    ln -s ${aptlyCompletionSrc}/aptly $out/share/bash-completion/completions
```

Comment:
Based on discussion here: https://github.com/NixOS/nixpkgs/issues/21053 completions should go to `bin` output, so please change to ` $bin/share/bash-completion/completions/`. I'm sorry for confusion.